csv2docx/csv2docx.py

```diff
@@ -1,12 +1,47 @@
 import csv
 from mailmerge import MailMerge
-def convert(data, template, delimiter=";"):
-    print ("Getting .docx template and .csv data files ...")
+from pathlib import Path
+
+
+def create_output_folder(path: str) -> Path:
+    """Creates a path to store output data if it does not exists.
+    Args:
+        path: the path from user in any format (relative, absolute, etc.)
+    Returns:
+        A path to store output data.
+    """
+    path = Path(path)
+    if not path.exists():
+        path.mkdir(parents=True, exist_ok=True)
+    return path
+
+
+def create_unique_name(filename: str, path: Path) -> Path:
+    """Checks whether file with same name exists or not
+    Args:
+        filename: the name of file to create
+        path: the path where the file is stored
+    Returns:
+        An unique full path with directory.
```

Comment:
Can you update the doc of 'Returns' to read:

- `The absolute path to the filename.`